pubspec.yaml

```diff
@@ -6,9 +6,9 @@ homepage: https://github.com/OneSignal/OneSignal-Flutter-SDK
 
 flutter:
   plugin:
-    androidPackage: com.onesignal.flutter
     platforms:
       android:
+        package: com.onesignal.flutter
```

Comment:
Compiler was complaining about using an outdated structure for declaring the androidPackage so updated it to the one it wanted here. Not critical for the purpose of this PR

Reply:
Thanks for the contribution! LGTM will recheck this on the next release, which will be on the following days